shell/src/main/java/tachyon/shell/TfsShell.java

```diff
@@ -138,8 +138,8 @@ public int run(String... argv) {
     try {
       command.run(args);
       return 0;
-    } catch (IOException ioe) {
-      System.out.println(ioe.getMessage());
+    } catch (IOException e) {
+      System.out.println(e.getMessage());
```

Comment:
I would just use the `LoggerFactory.getLogger(Constants.LOGGER_TYPE)` logger.

This is what other parts of the client-side code to (e.g. FileInStream).